CHANGELOG.md

```diff
@@ -50,5 +50,6 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * (ci) [\#457](https://github.com/line/lbm-sdk/pull/457) add swagger check
 * (ci) [\#469](https://github.com/line/lbm-sdk/pull/469) publish docker image on tag push
+* (ci) [\#471](https://github.com/line/lbm-sdk/pull/471) disable swagger check temporarily
```

Comment:
I think this changes is a fix for #457. 
Then, how about adding it to line 51 like below?
```
* (ci) #457, #471 add swagger check
```

Reply:
I agree on squashing the entries as you suggested. But the check would not working after this PR. Let me change the title.

Reply:
Is it OK to have different title with it's PR's? Or I cannot change the title because it affects the previous PR (#457).